src/db/config/config.js

```diff
@@ -3,29 +3,18 @@ const dotenv = require('dotenv');
 
 dotenv.config();
 
-const {
-  DB_USER,
-  DB_PASSWORD,
-  DB_NAME,
-  DB_HOST,
-  DB_PORT,
-  DB_DIALECT,
-  LOGGING,
-} = process.env;
+const { LOGGING } = process.env;
+const dialect = 'postgres';
 
 module.exports = {
   development: {
-    username: DB_USER,
-    password: DB_PASSWORD,
-    database: DB_NAME,
-    host: DB_HOST,
-    port: DB_PORT,
-    dialect: DB_DIALECT
+    use_env_variable: 'DATABASE_URL',
+    dialect
   },
   test: {
-    use_env_variable: 'DATABASE_URL',
-    dialect: 'postgres',
-    logging: LOGGING === 'true' && true
+    use_env_variable: 'DATABASE_URL_TEST',
```

Comment:
Hi Luqman, I like how clean this looks now, but I need to understand what is going on in here. Could you explain what you did here?

Reply:
Sure thing. I have refactored the file from using five (5) config properties to just one. It's a waste of space and more setup for someone new to the codebase.

Now all we have to do in `.env` is add this line
```
DATABASE_URL=postgresql://[host]/[databaseName]
DATABASE_URL_TEST=postgresql://[host]/[databaseName]
```

Reply:
OK, that makes sense! Good job.